pages/api/transport/[id].js

```diff
@@ -9,17 +9,17 @@ export default async function handler(req, res) {
 
   switch (method) {
     case "GET":
-      // request for get all infos about one transport : transport,driver,travelers
-      const id = req.query.id;
-
       try {
+        const id = req.query.id;
         const transport = await Transport.findById(id);
 
+        // a discuter peut etre le catch suffit
         if (!transport)
           return res
             .status(400)
             .json({ success: false, error: "Not match for this Id!" });
 
+        // A deplacer dans le Get Server Side Props
```

Comment:
Please add todo keyword :)

Reply:
i ve delete this snippet from BackEnd and send itself to Front-End .

Reply:
Okay even better :)